app/assets/javascripts/modules/track_search.js

```diff
@@ -0,0 +1,24 @@
+(function(Modules) {
+  "use strict";
+
+  Modules.TrackSearch = function() {
+    var that = this;
+
+    that.start = function(container) {
+
+      var filterSearchInput = container.find('table'),
```

Comment:
I've renamed tableInput to filterSearchInput and removed the original var: 
`filterSearchInput = container.find('.js-filter-table-input');`

I've replaced `filterSearchInput.on('keyup', '.js-filter-table-input', trackSearch);` with `container.on('keyup', '.js-filter-table-input', trackSearch);`

However, I still need the new `filterSearchInput` in order to get the text being searched for, so I've kept that variable.